features.bzl

```diff
@@ -40,6 +40,10 @@ _docs = struct(
     # The stardoc output changed in https://github.com/bazelbuild/bazel/commit/bd1c3af2ea14e81268e940d2b8ba5ad00c3f08d7
     # This may be required for "diff tests" that assert on the generated API docs.
     kwargs_name_with_double_star = ge("8.0.0-pre.20240603.2"),
+    # Starting with Bazel 8.1.0, all input to Stardoc is interpreted as UTF-8.
```

Comment:
Could you make this a bit more precise? What is `input` in this case? Assuming that doc comments have always been UTF-8, it's the Stardoc proto output that changes in 8.1.0 and I'm not even sure whether it's considered public API.

Reply:
> Assuming that doc comments have always been UTF-8

Definitely not, this would never have worked out of the box because it always resulted in double-encoding.

> the Stardoc proto output that changes in 8.1.0

What would that mean? Proto strings have always been UTF-8.

> I'm not even sure whether it's considered public API.

Pretty sure it is, starlark_doc_extract is publicly documented, the proto_library rule for stardoc_output.proto is public, etc.
The double-encoding would even happen without looking at the intermediary proto file, as it's always generated and fed to the Stardoc renderer.